source/funkin/play/character/AnimateAtlasCharacter.hx

```diff
@@ -131,7 +131,7 @@ class AnimateAtlasCharacter extends BaseCharacter
   {
     trace('[ATLASCHAR] Loading sprite atlas for ${characterId}.');
 
-    var sprite:FlxAtlasSprite = new FlxAtlasSprite(0, 0, Paths.animateAtlas(_data.assetPath, 'shared'));
```

Comment:
For this move you should rewrite Paths atlas thing since it takes the atlas without checking the shared library